content/en/docs/languages/java/exporters.md

```diff
@@ -104,22 +104,7 @@ import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
 
 import io.opentelemetry.api.OpenTelemetry;
-import io.opentelemetry.api.common.Attributes;
```

Comment:
This text above would need to be adjusted to reflect this change:

> In the case of [manual configuration] you can update the
[example app](/docs/languages/java/instrumentation#example-app) like the
following:

There are similar manual configuration snippets below for prometheus and zipkin that should be updated as well if this is the direction we want to go.

Reply:
Yes, I have that planned. Let me make these changes and then you can review. Thanks for prompt reply.